### PR TITLE
[4.0] Remove component sidebar from com_content

### PR DESCRIPTION
**Pull request by ciar4n**
Pull Request for Issue # .

### Summary of Changes
This component sidebar items for articles are identical to the main nav items. This PR removes the component sidebar.

### Testing Instructions
Navigate through all items under 'Content' and ensure the sidebar is removed and table displaying full width. Check that other list views are uneffected.

### Before
content-sb1

### After
content-sb2

### CODE REVIEW

**Comment by laoneo** [comment]
Do you plan to do the rest as well like com_contact?
image

**Comment by ciar4n** [comment]
I figured only components where the component sidebar is identical to the main navigation. Where there is a difference (eg. com_contacts) the sidebar can remain.

**Comment by franz-wohlkoenig** [comment]
I have tested this item :white_check_mark: successfully on a43eb9072cfae35020e804f8ddcaba139e965644<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/16386">issues.joomla.org/tracker/joomla-cms/16386</a>.</sub>

**Comment by laoneo** [comment]
So I guess we need to find out why there is a difference on the other components. Personally I think it should be avoided to have two menus there.

**Comment by brianteeman** [comment]
the only reason they are different (in core) was because we didnt insist that the sidebar was the same as the submenu.

with the new ui then that argument is mute and for contacts as an example everything in the sudebar can be moved to the menu

**Comment by brianteeman** [comment]
I have tested this item :white_check_mark: successfully on a43eb9072cfae35020e804f8ddcaba139e965644<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/16386">issues.joomla.org/tracker/joomla-cms/16386</a>.</sub>

**Comment by Bakual** [comment]
It's usually the fields links that are additionally in the sidebar, but are not in the menu.
This is because those links have a conditional (can be turned off in the components options) and that doesn't work for the menuitems.
So if we drop the parameter for the fields links, we can add those links to the menu and get rid of the sidebar in most components. Or we add the possibility for menuitems with conditionals. Personally I'd prefer the first option. :smile:

**Comment by brianteeman** [comment]
> This is because those links have a conditional (can be turned off in the components options) and that doesn't work for the menuitems.

check content in 3.7 - disable fields and they are gone from the menu :) 

or is that because of the hardcoded nature of the content menu where as components is more dynamic

**Comment by Bakual** [comment]
Yep, the content one is different. That's the only place where it works due to its hardcoded nature.

**Comment by ciar4n** [comment]
Conflicts fixed

File: administrator/components/com_content/Helper/ContentHelper.php [hunk 1]
```diff
@@ -63,6 +65,7 @@ public static function addSubmenu($vName)
 	}
 
 	/**
+>>>>>>> 68d9ed6adfd0a3fefd298ada86f3ea8bf07eaea2
```
Comment (Bakual) [inline administrator/components/com_content/Helper/ContentHelper.php]: @ciar4n That looks like an unsolved conflict here. As well on line 24/25

Comment (ciar4n) [inline administrator/components/com_content/Helper/ContentHelper.php]: Thank you @Bakual Fixed.

**Pull request merged**